### ISSUE TITLE
ModuleNotFoundError: No module named 'src.gen'

### ISSUE THREAD

**Issue by yangboz** [h2oai/h2ogpt#439]
after pip install -r requirements.tex then  running "python generate.py --base_model='llama' --prompt_type=wizard2 --score_model=None" 

env info 
M1 OS13.4,ython3.9 

it throws as following:

Traceback (most recent call last):
  File "/Users/yangboz/git/yangboz/h2ogpt/generate.py", line 8, in <module>
    from src.gen import main
ModuleNotFoundError: No module named 'src.gen'

any idea?

**Comment by pseudotensor** [comment]
Are you inside the h2ogpt directory when you run generate?

Can you try:

```
PYTHONPATH=".:src" python generate.py --base_model='llama' --prompt_type=wizard2 --score_model=None
```

**Comment by pseudotensor** [comment]
@Mathanraj-Sharma Do you hit this issue?

**Comment by pseudotensor** [comment]
FYI I just ran same exact command you ran on my MAC (no M1) and had no issues.

**Comment by Mathanraj-Sharma** [comment]
@pseudotensor I haven't hit the above issue on Mac M1 for both (Python 3.9 and 3.10).  

But I got a different error when I try with python 3.9
```bash
python generate.py --base_model='llama' --prompt_type=wizard2 --score_model=None
```

Error:
```bash
AttributeError: module 'types' has no attribute 'NoneType'
```

Because there is no `NoneType` in `types` python 3.9

<img width="668" alt="image" src="https://github.com/h2oai/h2ogpt/assets/30664910/118d3b89-5fba-4f2f-a6c3-27f830edb194">

<img width="569" alt="image" src="https://github.com/h2oai/h2ogpt/assets/30664910/c04cc5e0-dc71-43d8-8a4a-f2d960eb8b31">

@pseudotensor if you want I can try and send a fix for this NoneType issue for python 3.9

---
Things working fine as expected for `Python 3.10`. 

@yangboz please try with a clean environment, or please attach the exact steps you used to reproduce it at my end.
